databricks/koalas/series.py

```diff
@@ -465,6 +465,8 @@ def rsub(self, other):
     def mod(self, other):
         return (self % other).rename(self.name)
 
+    mode = mod
```

Comment:
@itholic, is `mode` same as `mod`? Seems not:

```python
>>> pdf.a.mode(1)
0    1
1    2
2    3
dtype: int64
>>> pdf.a.mod(1)
0    0
1    0
2    0
Name: a, dtype: int64
```

Reply:
You can remove it here and implement this later in a separate PR.

Reply:
@HyukjinKwon Oops, sorry it was mistaken. i've roll back this change